models/oakland_candidate.rb

```diff
@@ -30,6 +30,29 @@ def as_json(options = nil)
       last_name: last_name,
       ballot_item: office_election.id,
       office_election: office_election.id,
+
+      # contribution data
+      supporting_contribution_data: {
+        contributions_received: calculation(:total_contributions).try(:to_f),
+        total_contributions: calculation(:total_contributions).try(:to_f),
+        total_expenditures: calculation(:total_expenditures).try(:to_f),
+        total_loans_received: calculation(:total_loans_received).try(:to_f),
+        contributions_by_type: calculation(:contributions_by_type) || {},
+        expenditures_by_type: calculation(:expenditures_by_type) || {},
+      },
+      opposing_contribution_data: {
```

Comment:
Oh, I'm trying to get clarification from Suzanne, but I'm not sure that there is anyway for money opposing a candidate to be filed... like it would just show up as expenditures from another committee, but not actually tied in anyway to the candidate through filings... Do you know how that works?